agent-tooling/src/main/java/io/opentelemetry/auto/tooling/TracerInstaller.java

```diff
@@ -48,6 +48,10 @@ public static synchronized void installAgentTracer() {
     } else {
       log.info("Tracing is disabled.");
     }
+
+    if (Config.get().getPropagators() != null) {
+      PropagatorsInitializer.initializePropagators(Config.get().getPropagators());
+    }
```

Comment:
null check is not needed, `Config.getListSettingFromEnvironment` always returns non-null
```suggestion
    PropagatorsInitializer.initializePropagators(Config.get().getPropagators());
```